src/voodoo-gen/bin/main.ml

```diff
@@ -103,11 +103,26 @@ let generate_pkgver output name_filter version_filter =
             let otherdocs = files.otherdocs in
             let files = files.odocls in
             Format.eprintf "Found %d files\n%!" (List.length files);
+            let output file_path =
+              let p, ext = Fpath.split_ext ~multi:true file_path in
+              let extensions = String.split_on_char '.' ext |> List.rev in
+              let filename = Fpath.filename p |> String.uppercase_ascii in
+              let output_path =
+                match (filename, extensions) with
+                | "README", "html" :: _ -> Fpath.set_ext "html" p ~multi:true
+                | "README", "json" :: "toc" :: _ ->
+                    Fpath.set_ext "toc.json" p ~multi:true
+                | _ -> file_path
```

Comment:
What if there are multiple `README` files with a different extension? Currently, the last processed readme file will win. Should we prioritize based on the extension?